postinstall.one

```diff
@@ -32,7 +32,7 @@ then
 fi
 
 if [ -f /etc/init.d/one-context ]; then
-    if [ -d /etc/sysconfig/network-scripts ]; then
+    if [ -d /etc/sysconfig/network-scripts ] || [ -d /etc/sysconfig/network ]; then
```

Comment:
Yes, these are packages for the last release, but the current master is much different. Development packages can be created by the generate scripts in the repository root.

We still support CentOS 6, that's the reason for those lines. Your change doesn't break anything but also doesn't fix anything. Maybe the reason could be SLES11, but I don't think there are many people using such an old system in the cloud. OK, please leave the change, maybe it'll be useful one day.

Thank you.